### PR TITLE
[velero] Adding scheme and tlsConfig values to serviceMonitor template

### LINKED ISSUE #403: ServiceMonitor doesn't allow scheme and tlsConfig values
**Describe the problem/challenge you have**
Current ServiceMonitor template doesn't allow for `scheme` or `tlsConfig` options which are part of the ServiceMonitor CRD.

**Describe the solution you'd like**
Allow for optional specification and configuration of `metrics.serviceMonitor.scheme` & `metrics.serviceMonitor.tlsConfig`

**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]

**Environment:**

- helm version (use `helm version`): 3.9.4
- helm chart version and app version (use `helm list -n <YOUR NAMESPACE>`): 2.31.3 | 1.9.1
- Kubernetes version (use `kubectl version`): 1.25
- Kubernetes installer & version: RKE2
- Cloud provider or hardware configuration: AWS
- OS (e.g. from `/etc/os-release`): RHEL

### PR DESCRIPTION
**Pull request by rgarcia6520**
Signed-off-by: garcia.ryan <garcia.ryan@solute.us>

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the values.yaml or README.md
- [x] Title of the PR starts with chart name (e.g. `[velero]`)

Closes #403

### CODE REVIEW

File: charts/velero/values.yaml [hunk 1]
```diff
@@ -148,10 +148,15 @@ metrics:
     prometheus.io/path: "/metrics"
 
   serviceMonitor:
-    enabled: false
+    enabled: true
```
Comment (jenting) [inline charts/velero/values.yaml:151]: Please default to false to keep the backward compatible.

File: charts/velero/values.yaml [hunk 2]
```diff
@@ -148,10 +148,15 @@ metrics:
...
     additionalLabels: {}
     # ServiceMonitor namespace. Default to Velero namespace.
     # namespace:
+    # ServiceMonitor connection scheme. Defaults to HTTP.
+    scheme: "https"
```
Comment (jenting) [inline charts/velero/values.yaml:156]: Since the default is HTTP, why do we put it as HTTPS?

File: charts/velero/values.yaml [hunk 3]
```diff
@@ -148,10 +148,15 @@ metrics:
...
+    # ServiceMonitor connection tlsConfig. Defaults to {}.
+    tlsConfig:
+      caFile: /etc/ssl/HAHA.pem
```
Comment (jenting) [inline charts/velero/values.yaml:159]: Please use the default value here. E.g.,
`tlsConfig: {}`

**Review by jenting — CHANGES_REQUESTED** [review]

**Comment by rgarcia6520** [comment]
@jenting Apologies, accidentally had comitted test values I put in there to ensure things templated correctly, values file is updated

**Comment by jenting** [comment]
> @jenting Apologies, accidentally had comitted test values I put in there to ensure things templated correctly, values file is updated

Thank for your contribution.

If you want to test against CI, you could change the value under ci folder.

**Review by jenting — APPROVED** [review]

**Review by sseago — APPROVED** [review]

**Pull request merged**